site/content/blog/2025/09/19/apache-polaris-1.1.0-incubating.md

```diff
@@ -0,0 +1,47 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+title: Apache Polaris 1.1.0-incubating has been released!
+date: 2025-09-19
+---
+
+The Apache Polaris team is pleased to announce Apache Polaris 1.1.0-incubating.
+
+This release includes:
+* **New features & enchancements**
+** Support for non-AWS S3 compatible storage with STS
+** Use of Realm instead of RealmId
+** Documentation improvements (release process, multi-realms configuration)
+* **Bug fixes**
+** Fix drop view with default server configuration
+** Fix MinIO support
+** Remove ThreadLocal
+
+This release can be downloaded:
+* https://polaris.apache.org/downloads/
+
+The artifacts are available on Maven Central.
+
+The Docker images are available on Docker Hub:
+* https://hub.docker.com/r/apache/polaris/tags
```

Comment:
FWIW I don't. see `1.1.0-incubating` tag on Docker Hub. But I do see a `1.1.0-incubating-SNAPSHOT` tag that is 3 months old, and imho it should be removed.

Reply:
The upload is on going.

Reply:
Done